base/intfuncs.jl

```diff
@@ -748,12 +748,15 @@ split_sign(n::Unsigned) = n, false
 Convert an integer `n` to a string in the given `base`,
 optionally specifying a number of digits to pad to.
 
+See also [`digits`](@ref), [`bitstring`](@ref).
```

Comment:
Should you add `count_zeros` and `count_ones`?

Reply:
Oh I didn't know `count_zeros` existed. I've added it here and to `bitstring`. There's a whole zoo of functions like `trailing_zeros` but perhaps I should stop.